transport/http_test.go

```diff
@@ -29,6 +30,7 @@ func init() {
 	os.Unsetenv("ELASTIC_APM_SERVER_URLS")
 	os.Unsetenv("ELASTIC_APM_SERVER_URL")
 	os.Unsetenv("ELASTIC_APM_SECRET_TOKEN")
+	os.Unsetenv("ELASTIC_APM_SECRET_CERT")
```

Comment:
should probably be `ELASTIC_APM_SERVER_CERT`?

Reply:
oops, thanks!